packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -121,6 +121,63 @@ import {Text} from '@react-spectrum/text';
 </MenuTrigger>
 ```
 
+## Unavailable Items <VersionBadge version="alpha" style={{marginLeft: 4, verticalAlign: 'bottom'}} />
+
+ContextualHelpTrigger allows making a Menu Item unavailable along with showing a user extra information about the state of that unavailable item.
+It provides context and may point users to more information elsewhere.
+
+The ContextualHelpTrigger accepts exactly
+two children: the Item which triggers opening of the Dialog and the Dialog itself. The trigger must be
+the first child passed into the ContextualHelpTrigger and should be an Item. Similar to
+ContextualHelp, the layout of the Dialog is very deliberate. See [ContextualHelp](ContextualHelp.html#content) for further explanation.
+
+By default, a ContextualHelpTrigger's Dialog is opened by hovering, pressing the trigger element or activating
+it via the <Keyboard>Space</Keyboard>, <Keyboard>Enter</Keyboard>, or <Keyboard>Right Arrow</Keyboard> keys.
+Hovering another item or pressing the <Keyboard>Esc</Keyboard> key will close the Dialog and leave the Menu open.
+
+When Menu Items are unavailable via a ContextualHelpTrigger, the Menu `onAction` and `onSelectionChange`
+callbacks will not occur.
```

Comment:
nit: maybe something like this? It previously sounded like onAction/onSelectionChange wouldn't fire for any of the items even if 
```suggestion
Note that the Menu's `onAction` and `onSelectionChange`
callbacks will not fire for items made unavailable by a ContextualHelpTrigger.
```

Reply:
Or maybe move this sentence after the one below (aka the "Setting the `isUnavailable` prop on the...")?